ts/tests/unit_tests/test_decorator.py

```diff
@@ -0,0 +1,37 @@
+from ts.utils.serve_decorator import serve, torchserve_start, torchserve_stop, archive_model, create_handler, create_torchserve_config
+from ts.torch_handler.base_handler import BaseHandler
+import os
+
+class ToyHandler(BaseHandler):
+    def inference(self):
+        return "42"
+
+def test_create_handler():
+    create_handler(ToyHandler)
+    assert os.path.exists("handler.py")
+
+def test_archive_model():
+    # TODO: Use a real model.pt
+    with open("model.pt", "w") as f:
+        pass
+    assert os.path.exists("model.pt")
+    archive_model(serialized_file="model.pt", handler="handler.py")
+    assert os.path.exists(os.path.join("model_store", "model.mar"))
+
+def test_create_torchserve_config():
+    create_torchserve_config()
+    assert os.path.exists("config.properties")
+
+def test_start():
+    error = torchserve_start(create_handler(ToyHandler))
+    assert error == 0
+    error = torchserve_stop()
+    assert error == 0
+
+def test_integration():
```

Comment:
@msaroufim it will be helpful to have a test/ example that showcase the decorator part.

Reply:
The test for the decorator is `serve(ToyHandler)`. A decorator is nothing but a function that takes in as argument the function or class below it